docs/how-to-guides/azure-deployment-cli.md

```diff
@@ -112,10 +112,6 @@ az group create -l $location -n $resource_group_name
 sp_password=$(az ad sp create-for-rbac --name $service_principal_name --role Contributor --scopes /subscriptions/$subscription_id/resourceGroups/$resource_group_name --query "[password]" --output tsv)
 sp_appid=$(az ad sp list --display-name $service_principal_name --query "[].{appId:appId}" --output tsv)
 sp_tenantid=$(az ad sp list --display-name $service_principal_name --query "[].{appOwnerOrganizationId:appOwnerOrganizationId}" --output tsv)
-echo "AZURE_CLIENT_ID: $sp_appid"
-echo "AZURE_TENANT_ID: $sp_tenantid"
-echo "AZURE_CLIENT_SECRET: $sp_password"
-This will give three variables: AZURE_CLIENT_ID, AZURE_TENANT_ID and AZURE_CLIENT_SECRET. You will need them later.
```

Comment:
@jainr i think all content under Create a Service Principal which you can automate later in this doc can be removed, what do you think? I remember feathr deplyoment is already switched to managed identity.

Reply:
Do we have any solutions here? Shall we keep all of those credentials for now?

Reply:
This is CLI way of running things. We have automated it in ARM.

Reply:
For ARM, Service Principal is no longer used and replaced with managed identity. The question here is service pripincal still needed for CLI deployment?

Reply:
Hold this change for now to avoid issues.

Reply:
The CLI doc still uses SP and assign roles to it for Blob, Synapse and so on. If we remove this section the user won't be able to know the values of it.

Reply:
I suggest keeping it, or if you plan to remove it, remove it from following sections too. @enya-yx

Reply:
I see. Will keep it for now.